### PR TITLE
Add GitHub Actions support

### PR DESCRIPTION
**Pull request by traversaro**
The test always test with the latest vcpkg to quickly detect regressions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Run on latest vcpkg 
+      shell: bash
+      run: |
+        mkdir C:/vcpkg-test
+        git clone https://github.com/microsoft/vcpkg  C:/vcpkg-test/vcpkg
+        C:/vcpkg-test/vcpkg/bootstrap-vcpkg.sh
+        C:/vcpkg-test/vcpkg/vcpkg.exe --overlay-ports=${GITHUB_WORKSPACE} install --triplet x64-windows ipopt-binary
```
Comment (pattacini) [inline .github/workflows/ci.yml:33]: Don't we need to install yet a further library after ipopt in order to test the full functionality?

Comment (traversaro) [reply .github/workflows/ci.yml:33]: Done in https://github.com/robotology/robotology-vcpkg-binary-ports/pull/5/commits/81a4a37dcf0685202dd0efc6e202f45d3cbfe85f .

**Review by pattacini — APPROVED** [review]

**Comment by traversaro** [comment]
Upstream issue that is causing the CI failure: https://github.com/microsoft/vcpkg/issues/10119 .

**Comment by traversaro** [comment]
As at the moment the master branch is not working due to https://github.com/microsoft/vcpkg/issues/10119, I think we can just test against the latest two releases (2020.01 and 2019.12) and add the test against master when https://github.com/microsoft/vcpkg/issues/10119 is fixed.

**Pull request merged**